ConferencesApp/ConferenceDataStore.swift

```diff
@@ -88,6 +90,51 @@ class ConferenceDataStore {
         }
     }
     
+    func updateCalendarEventIdentifier( conf:Conference, eventId:String) {
+
+        let realm = try! Realm()
+
+        do {
+            try realm.write {
+                let calEvent =  CalendarEventData()
+                calEvent.id = conf.id
+                calEvent.eventID = eventId
+                
+                realm.create(CalendarEventData.self, value: calEvent, update: true)
+           
+                print("Updated calenderEventID")
+                debugPrint(calEvent)
+            }
+        } catch {
+                print("Could not save calendar event ID in database.")
+        }
+    }
+    
+    func deleteCalendarEventData(conf:Conference) -> Void {
```

Comment:
Colon Violation: When specifying a type, always associate the colon with the identifier